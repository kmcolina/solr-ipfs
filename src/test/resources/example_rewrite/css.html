<?xml version="1.1" encoding="iso-8859-1"?>
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="da">
<head>
    <title>kimse.rovfisk.dk/katte/</title>
    <style type="text/css" media="screen">@import "/css/forum_o1.css";</style>
    <meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1" />
    <link rel="stylesheet" href="/style_o2.css"  type="text/css" media="screen" />
    <link rel="alternate" type="application/rss+xml" title="RSS 2.0" href="/rss_o3.php" />
</head>
<body>
<style type="text/css" media="screen">@import "//www.dr.dk/drdkGlobal/spot/spotGlobal_o4.css";</style>
<style type="text/css" media="screen">@import "/design/www/global/css/globalPrint_o5.css";</style>
<style type="text/css" media="screen">@import url(http://en.statsbiblioteket.dk/portal_css/SB%20Theme/resourceplonetheme.sbtheme.stylesheetsmain-cachekey-7e976fa2b125f18f45a257c2d1882e00_o6.css);</style>
<!-- Note: In 1.14.2, JSoup changes from preserving the first class attribut to preserving the last, so "button" is removed -->
<a class="toplink" href="test" class="button" style="background:url(img/homeico_o7.png) no-repeat ; width:90px; margin:0px 0px 0px 16px;">kimse.rovfisk.dk  </a>
<a class="toplink" href="/katte/">katte / </a><br /><br />
<table cellspacing="8"><tr><td></td><td class="itemw"><a href="/katte/?browse=DSC00175.JPG"><img class="lo" src="/cache/katte/DSC00175_o8.JPG" /></a></td>
    <td></td><td class="itemw"><a href="/katte/?browse=DSC00209.JPG"><img class="lo" src="/cache/katte/DSC00209_o9.JPG" /></a></td>
    <img  src="http://belinda.statsbiblioteket.dk:9721/solrwayback/services/downloadRaw?source_file_path=/netarkiv/0122/filedir/285330-279-20180113095114447-00002-kb-prod-har-001_o10.kb.dk.warc.gz&amp;offset=685207223" srcset="http://skraedderiet.dk/wp-content/uploads/2018/01/saks_watermark-001_transparent_o11.jpg 477w, http://skraedderiet.dk/wp-content/uploads/2018/01/saks_watermark-001_transparent-150x150_o12.jpg 150w, http://skraedderiet.dk/wp-content/uploads/2018/01/saks_watermark-001_transparent-298x300_o13.jpg 298w" sizes="(max-width: 235px) 100vw, 235px">
</table><br />  </body>
</html>"