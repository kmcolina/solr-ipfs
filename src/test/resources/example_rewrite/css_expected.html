<!--?xml version="1.1" encoding="iso-8859-1"?-->
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="da">
 <head>
  <title>kimse.rovfisk.dk/katte/</title>
  <style type="text/css" media="screen">@import "http://localhost:0000/solrwayback/services/downloadRaw?source_file_path=somesourcefile&offset=1";</style>
  <meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
  <link rel="stylesheet" href="http://localhost:0000/solrwayback/services/view?source_file_path=somesourcefile&amp;offset=2" type="text/css" media="screen">
  <link rel="alternate" type="application/rss+xml" title="RSS 2.0" href="http://localhost:0000/solrwayback/services/view?source_file_path=somesourcefile&amp;offset=3">
 </head>
 <body>
  <style type="text/css" media="screen">@import "http://localhost:0000/solrwayback/services/downloadRaw?source_file_path=somesourcefile&offset=4";</style>
  <style type="text/css" media="screen">@import "http://localhost:0000/solrwayback/services/downloadRaw?source_file_path=somesourcefile&offset=5";</style>
  <style type="text/css" media="screen">@import url(http://localhost:0000/solrwayback/services/downloadRaw?source_file_path=somesourcefile&offset=6);</style>
  <!-- Note: In 1.14.2, JSoup changes from preserving the first class attribut to preserving the last, so "button" is removed -->
  <a class="toplink" href="http://localhost:0000/solrwayback/services/web/20200430130700/http://example.com/somefolder/test" style="background:url(http://localhost:0000/solrwayback/services/downloadRaw?source_file_path=somesourcefile&amp;offset=7) no-repeat ; width:90px; margin:0px 0px 0px 16px;">kimse.rovfisk.dk </a>
  <a class="toplink" href="http://localhost:0000/solrwayback/services/web/20200430130700/http://example.com/katte/">katte / </a>
  <br>
  <br>
  <img src="http://localhost:0000/solrwayback/services/downloadRaw?source_file_path=somesourcefile&amp;offset=10" srcset="http://localhost:0000/solrwayback/services/downloadRaw?source_file_path=somesourcefile&amp;offset=11 477w, http://localhost:0000/solrwayback/services/downloadRaw?source_file_path=somesourcefile&amp;offset=12 150w, http://localhost:0000/solrwayback/services/downloadRaw?source_file_path=somesourcefile&amp;offset=13 298w" sizes="(max-width: 235px) 100vw, 235px">
  <table cellspacing="8">
   <tbody>
    <tr>
     <td></td>
     <td class="itemw"><a href="http://localhost:0000/solrwayback/services/web/20200430130700/http://example.com/katte/?browse=DSC00175.JPG"><img class="lo" src="http://localhost:0000/solrwayback/services/downloadRaw?source_file_path=somesourcefile&amp;offset=8"></a></td>
     <td></td>
     <td class="itemw"><a href="http://localhost:0000/solrwayback/services/web/20200430130700/http://example.com/katte/?browse=DSC00209.JPG"><img class="lo" src="http://localhost:0000/solrwayback/services/downloadRaw?source_file_path=somesourcefile&amp;offset=9"></a></td>
    </tr>
   </tbody>
  </table>
  <br>  "
 </body>
</html>